saleor/graphql/checkout/mutations.py

```diff
@@ -835,6 +835,8 @@ def perform_mutation(cls, _root, info, checkout_id, store_source, **data):
                 tracking_code=tracking_code,
                 redirect_url=data.get("redirect_url"),
             )
+        if info.context.site.settings.automatically_confirm_all_new_orders:
```

Comment:
The saleor.order module has file `actions`. The file defines all mandatory order events like `order_created`, `order_cancelled`, `order_fully_paid` etc. 
We should create there the `order_confirmed` action. I assume that this function will trigger the `plugins.order_confirmed(order)` event, but what about the `OrderEvents`? Don't we want anything about it in order history?
If you add `order_confirmed` to `actions`, you could move this `if` to action `order_created` and trigger `order_confirmed` when `automatically_confirm_all_new_orders` is enabled.
For `OrderConfirm` mutation you could only trigger the action -`order_confirmed`